### ISSUE TITLE
[REQUEST] SOCKS5 Proxy with PAM authentification

### ISSUE THREAD

**Issue by FallenKN** [saenzramiro/rambox#740]
It would be nice to use a proxy with authentification and changing the proxy mode to for example SOCKS5 or HTTP..

**Comment by henriquedesousa** [comment]
Yes, I'm having the same issue, need proxy with authentication. Simply putting `http://username:password@url` doesn't work.

**Comment by benavern** [comment]
+1

**Comment by fuelflo** [comment]
+1
this would be much appreciated!

**Comment by tkrameral** [comment]
+1

**Comment by BORN2LOSE** [comment]
+1. Socks5 from SSH tunnel, for example:
socks5://127.0.0.1:9150

**Comment by gomaaz** [comment]
+1

**Comment by wokawoka** [comment]
+1
and, ideally, being able to potentially set a different proxy for any different service would be incredibly useful!

**Comment by aguerre** [comment]
+1
Cannot use Rambox anymore without it !

**Comment by GreatBigWhiteWorld** [comment]
Yes, desperately needing socks5 proxy support

**Comment by fuelflo** [comment]
That's true....one year has passed but unfortunately we are not a step closer... :(

**Comment by GreatBigWhiteWorld** [comment]
It's freeware so I wouldn't complain. Just wonder if it's hard to add the feature or the devs are just not interested?

**Comment by FallenKN** [comment]
It has been added as a milestone so just keep patience.
There are a few other things having more priority

**Comment by barfin** [comment]
+1
